runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg

```diff
@@ -1,21 +1,17 @@
 TestType() ::= "Lexer"
 
-Options ::= [
-	"Ignore": "Currently unable to generate the required CRLF input sequence."
-]
-
 Grammar ::= [
 	"L": {<grammar("L")>}
 ]
 
-Input() ::= "34<\u000D><\n> 34"
```

Comment:
one basic rule in unit testing is that a test should only serve one purpose
I would suggest you write a new test for just that: handling of carriage returns
and have those tests running since they cover other stuff

> Le 21 juin 2015 à 03:21, Sam Harwell <notifications@github.com> a écrit :
> 
> In runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/LexerExec/CharSet.stg <https://github.com/antlr/antlr4/pull/927#discussion_r32887173>:
> 
> >  Grammar ::= [
> >  	"L": {<grammar("L")>}
> >  ]
> >  
> > -Input() ::= "34<\u000D><\n> 34"
>  I'm planning to fix the handling of carriage returns to ensure the test suite includes coverage of these characters, which do not behave the same way as new line characters in ANTLR. Until it's working under the original design, I would prefer to not make any changes to this.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/927/files#r32887173>.
>